Modules/Microsoft365DSC/DSCResources/MSFT_EXOHostedContentFilterRule/MSFT_EXOHostedContentFilterRule.psm1

```diff
@@ -326,6 +326,7 @@ function Set-TargetResource
         $UpdateParams.Remove('CertificatePassword') | Out-Null
         $UpdateParams.Remove('ManagedIdentity') | Out-Null
         $UpdateParams.Remove('Enabled') | Out-Null
+        $UpdateParams.Remove('HostedContentFilterPolicy') | Out-Null
```

Comment:
Set-HostedContentFilterRule can still be called with -HostContentFilterPolicy passed in if it doesn't already have an associated rule. We should check to see if the policy has an associated rule and ensure it's not the default policy before deciding whether or not to remove the parameter.

Reply:
OK, I got your point, let me do a few tests